sdk/monitor/monitor-query/review/monitor-query.api.md

```diff
@@ -81,6 +59,12 @@ export interface LogsColumn {
 // @public
 export type LogsColumnType = string;
 
+// @public
+export interface LogsErrorInfo extends Error {
```

Comment:
technically don't we only need `code` here? `Error` is defined as:

```ts
interface Error {
    name: string;
    message: string;
    stack?: string;
}
```

I don't feel too strongly about this as redeclaring the properties isn't hurting anything, but I wanted to point it out

Reply:
Oh I thought code and message should be part of error returned by service

Reply:
Also is `stack` something that's defined internally? I am not assigning any values to `stack` though. Should we push the entire error object returned by the service into the stack?

Reply:
I removed the `message` property

Reply:
When you instantiate an error, stack will be defined on it. So if you push `BatchError` onto your failed results, each one should have a `stack` property.